api_tests/RegisterJetsonboxid_post_others.yaml

```diff
@@ -0,0 +1,71 @@
+# Test generated by RoostGPT for test qlue-single-api-test using AI Type Open AI and AI Model gpt-4
+
+
+# Test generated for /RegisterJetsonboxid_post for http method type POST in artillery framework
+
+# RoostTestHash=c528c38714
+
+
+config:
+  target: "{{ $processEnvironment.API_HOST }}"
+  plugins:
+    expect: {}
+  phases:
+    - duration: 60
+      arrivalRate: 20
+  payload:
+    path: "RegisterJetsonboxid_post_others.csv"
+    fields:
+      - "jetson_uuid"
+      - "username"
+      - "clusterId"
+      - "houseId"
+    skipHeader: true
+
+scenarios:
+  - name: "Non-success API responses"
+    flow:
+      # Invalid payload
+      - post:
+          url: "/RegisterJetsonboxid?jetson_uuid={{ jetson_uuid }}&username={{ username }}&clusterId={{ clusterId }}&houseId={{ houseId }}"
```

Comment:
This is getting correctly embedded in url for api test as opposed to integration test